### PR TITLE
Issue/#637

### PR DESCRIPTION
**Pull request by finbartracey1**
High-lights matching portion of project title with search term on dashboard. Also search box when it has a value.

### CODE REVIEW

File: frontend/src/core/dashboard/views/dashboardSidebarView.js [hunk 1]
```diff
@@ -29,10 +29,18 @@ define(function(require) {
             this.usedTags = [];
         },
 
+        highlightSearchBox: function(){
+                this.$('.dashboard-sidebar-filter-search-input').removeClass('search-highlight')
```
Comment (dennis-learningpool) [inline frontend/src/core/dashboard/views/dashboardSidebarView.js]: extra tab here

**Comment by dennis-learningpool** [comment]
+1

File: frontend/src/core/dashboard/views/dashboardView.js [hunk 2]
```diff
@@ -252,6 +253,11 @@ define(function(require){
     },
 
     appendProjectItem: function(projectModel) {
+      var output = '<span class="highlighted">'+ this.filterText+ '</span>';
```
Comment (finbartracey1) [inline frontend/src/core/dashboard/views/dashboardView.js]: this line needs removed

**Comment by brian-learningpool** [comment]
+1

**Pull request merged**